<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="aspire818" name="com.projectname.test.functional.demographics.encounterreferral.VerifyReferalByPractice" tests="1" failures="0" timestamp="13 May 2015 14:42:43 GMT" time="5662.084" errors="1">
  <testcase name="verifyReferalByPractice" time="5662.084" classname="com.projectname.test.functional.demographics.encounterreferral.VerifyReferalByPractice">
    <error type="java.lang.AssertionError" message="Couldn&#039;t find Referred By Provider expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Couldn't find Referred By Provider expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at com.projectname.testutils.seleniumutils.SeleniumWebDriver.failNotEquals(SeleniumWebDriver.java:739)
	at com.projectname.testutils.seleniumutils.SeleniumWebDriver.assertTrue(SeleniumWebDriver.java:733)
	at com.projectname.testutils.pages.ReferralsPage.verifyReferralDetails(ReferralsPage.java:528)
	at com.projectname.testutils.pages.DemographicsPage.verifyReferralDetails(DemographicsPage.java:233)
	at com.projectname.test.functional.demographics.encounterreferral.VerifyReferalByPractice.verifyReferalByPracticeOption(VerifyReferalByPractice.java:151)
	at com.projectname.test.functional.demographics.encounterreferral.VerifyReferalByPractice.verifyReferalByPractice(VerifyReferalByPractice.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- verifyReferalByPractice -->
</testsuite> <!-- com.projectname.test.functional.demographics.encounterreferral.VerifyReferalByPractice -->
